specs/commandline/policy.md

```diff
@@ -0,0 +1,232 @@
+            ],
+            "trustedIdentities": [
+                "x509.subject: C=US, ST=WA, L=Seattle, O=wabbit-networks.io, OU=Security Tools"
+            ]
+        },
+        {
+            "name": "unsigned-image",
+            "registryScopes": [ "registry.acme-rockets.io/software/unsigned/net-utils" ],
+            "signatureVerification": {
+              "level" : "skip" 
+            }
+        },
+        {
+            "name": "allow-expired-images",
+            "registryScopes": [ "registry.acme-rockets.io/software/legacy/metrics" ],
+            "signatureVerification": {
+              "level" : "strict",
+              "override" : {
+                "expiry" : "log"
+              }
+            },
+            "trustStores": ["ca:acme-rockets"],
+            "trustedIdentities": ["*"]
+        },
+        {
+            "name": "global-policy-for-all-other-images",
+            "registryScopes": [ "*" ],       
+            "signatureVerification": {                                
+                "level": "strict"
+            },
+            "trustStores": [ 
+              "ca:acme-rockets"
+            ],                  
+            "trustedIdentities": [                                    
+                "x509.subject: C=US, ST=WA, L=Seattle, O=acme-rockets.io, CN=SecureBuilder"
+            ]
+        }
+    ]
+}
+```
+
+## Outline
+
+### notation policy command
+
+```text
+Manage trust policies for signature verification.
+
+Usage:
+  notation policy [command]
+
+Available Commands:
+  import    import trust policies from a JSON file
+  init      initialize trust policies with default values
+  show      show trust policies
+
+Flags:
+  -h, --help   help for policy
+```
+
+### notation policy import
+
+```text
+Import trust policies from a JSON file
+
+Usage:
+  notation policy import [flags] <file_path>
+
+Flags:
+  -d, --debug     debug mode
+      --force     override the existing trust policies, never prompt
+  -h, --help      help for import
+  -v, --verbose   verbose mode
+```
+
+### notation policy init
+
+```text
+Initialize trust policies with default values
+
+Usage:
+  notation policy init [flags]
+
+Flags:
+  -d, --debug             debug mode
+      --force             override the existing trust polices, never prompt
+  -h, --help              help for export
+      --ts stringArray    specify the trust stores in format "<type>:<name>", e.g. "ca:my_store"
+  -v, --verbose           verbose mode
+```
+
+### notation policy show
+
+```text
+Show trust policies
+
+Usage:
+  notation policy [flags] show
+
+Flags:
+  -d, --debug     debug mode
+  -h, --help      help for show
+  -v, --verbose   verbose mode
+```
+
+## Usage
+
+### Initialize trust policies with default values
+
+```shell
+notation policy init
+```
+
+Upon successful execution, trust policies with default values are created as following. Make sure to update the `trustStores` property with correct values.
```

Comment:
I'm trying to understand the logic here. In this case, the user chooses to ignore the `--trust-store` flag, thus, the default `ca:default` trust store is used. If the user decided to go for the default, do they still need to "Make sure to update the `trustStores` property with correct values." ? Ideally, they don't need to modify the trust policy statement to get start.
```suggestion
Upon successful execution, trust policies with default values are created as following. Since the `--trust-store` flag is not specified, a default trust store `ca:default` is used.
```
Btw, we should print out the trust policy statement generated by `notation policy init` command.

Reply:
The default trust store `ca:default` may not exist, so need to inform users to create one or use other trust stores.

Reply:
> The default trust store `ca:default` may not exist, so need to inform users to create one or use other trust stores.

Actually, that's not true. After the user called `notation policy init`, the next step would be adding their certificates into the trust store, for example, `notation cert add -t ca -s default ./cert1.crt ./cert2.crt`, this command will automatically create directory `truststore/x509/ca/default` and add cert1.crt and cert2.crt into the trust store. (Since the default trust policy is printed out as the output of `notation policy init` command, the user would explicitly see `ca:default` as their trust store.)